packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -28,7 +28,8 @@ export const findOneQuery = (t: ModelDefinition) => {
     ${fieldName}(id: $id) {
       ...${t.graphqlType.name}ExpandedFields
     }
-  }`
+}
+`
```

Comment:
This fixes the original problem, so thank you. This also introduces an extra blank line too though:

![image](https://user-images.githubusercontent.com/11743717/94658591-e743ec80-02fa-11eb-995a-7b9688c979d5.png)

Reply:
@craicoverflow thanks for reviewing. This is happening for the generated mutations too, so should I fix this everywhere?
![ssToday](https://user-images.githubusercontent.com/56963264/94692720-7cbd9b80-0350-11eb-9bce-5117506d717d.png)

Reply:
Sure! If you see other area that are wrong you can fix those too and we can review the code.